website/index.html

```diff
@@ -106,13 +112,13 @@
   <!-- /container -->
   <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
   <script>
-    window.jQuery || document.write('<script src="/js/vendor/jquery-1.11.2.min.js"><\/script>')
+    window.jQuery || document.write('<script src="js/vendor/jquery-1.11.2.min.js"><\/script>')
```

Comment:
Just wanted to point out that this seems to be requiring jquery twice.

Reply:
Ah, I guess it's a fallback to the local copy if the CDN is down?

Web is weird :joy:

Reply:
This is well known hack to support file protocol (local dev)